integration-tests/src/test/resources/weather/cities.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Skoro cities to albo tablica jedno elementowa, albo city.json. Ale chyba cities wygodniejsze z punktu widzenia używania w testach